vue-app/src/utils/renderAddressOrHash.ts

```diff
@@ -0,0 +1,13 @@
+export function renderAddressOrHash(
```

Comment:
I don't believe this is needed anymore - @wackerow removed these in https://github.com/ethereum/clrfund/pull/314

Reply:
Actually never mind looks like we still use it in one place: https://github.com/ethereum/clrfund/blob/develop/vue-app/src/components/TransactionReceipt.vue#L75

Reply:
Ill move this method to the accounts file, and do a passover of where it will be used again.